pallets/settlement/src/lib.rs

```diff
@@ -246,6 +251,8 @@ pub struct Instruction<Moment, BlockNumber> {
     pub trade_date: Option<Moment>,
     /// Date after which the instruction should be settled (not enforced)
     pub value_date: Option<Moment>,
+    /// Memo of the instruction
+    pub instruction_memo: Option<InstructionMemo>,
```

Comment:
This field isn't needed, since the memos are stored in a separate storage map.  Adding this field  would require doing a migration.